examples/example-predeployed-connector/src/main.tsx

```
@@ -0,0 +1,13 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import "./index.css";
+import App from "./App.tsx";
+import StarknetProvider from "./starknet-provider.tsx";
+
+createRoot(document.getElementById("root")!).render(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)